blocked-edges/4.7.11.yaml

```diff
@@ -0,0 +1,3 @@
+to: 4.7.11
+from: .*
+# CMO goes degraded when user monitoring workflow is enabled due to DNS changes https://bugzilla.redhat.com/show_bug.cgi?id=1953518
```

Comment:
4.7.11 has the fix, so this is not correct. We need to block edges to versions before 4.7.11

Reply:
Not all versions before 4.7.11, so trying to find out which version.